llvm/test/Feature/Repo/Inputs/repo_contributions_dependencies_loop.ll

```diff
@@ -0,0 +1,30 @@
+target triple = "x86_64-pc-linux-gnu-elf"
+
+%"T1" = type { i8* }
+%"T2" = type { i32 (...)**, i32, i8* }
```

Comment:
I think we can simplify this test a little. The two types T1 and T2 aren't really needed: we can just stick to the fundamental types. The functions themselves can be reduced to just a call or a global variable access. This presence of an argument called 'this' in Func1 and Func2 suggests that the code was originally C++: are they necessary?

Reply:
Simplified the file in commit cca1f2d.